src/Block_Network.f90

```diff
@@ -21,4 +21,11 @@ Module Block_Network
     real   :: delta_n,n_default=2
     real   :: dt_comp
     real, dimension(:), allocatable  :: ndelta
+!
+!      Logical variables
+!
+    logical :: source  ! presence of thermal plant
+
+
+
```

Comment:
Delete the three blank lines here or just leave one. There's no need to put so many blank lines here